internal/configuration/settings/helpers/merge.go

```diff
@@ -139,13 +139,13 @@ func MergeWithHTTPHandler(existing, other http.Handler) (result http.Handler) {
 	return other
 }
 
-func MergeStringSlices(a, b []string) (result []string) {
+func MergeSlices[K string | uint16](a, b []K) (result []K) {
 	if a == nil && b == nil {
 		return nil
 	}
 
-	seen := make(map[string]struct{}, len(a)+len(b))
-	result = make([]string, 0, len(a)+len(b))
+	seen := make(map[K]struct{}, len(a)+len(b))
+	result = make([]K, 0, len(a)+len(b))
```

Comment:
Let's change `K` to `T` since from a caller perspective, `K` is not really used as a key in map, but only as the Type of elements of a slice.